### DISCUSSION TITLE
Unable to search my own repositories

### DISCUSSION THREAD

**Discussion by begin-again** [community/community#40836] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I select my self from the drop down box like so, and type a few characters in the search box it immediately switches to all repos
![image](https://user-images.githubusercontent.com/11179873/205661764-fd651d6a-f5a7-4978-9cc6-bbd199836ed5.png)
![image](https://user-images.githubusercontent.com/11179873/205662088-f04749d4-69e6-433e-b2fd-f7ddea1142dc.png)

all attempts to specify an `org` or `user` for self fail. I thought it might be the dash but when I enter in an org I belong to, `org:dorsett-tech` that works fine.
![image](https://user-images.githubusercontent.com/11179873/205662397-9cca8588-2dae-470e-9626-19ce26936060.png)

**Comment by ossanna16** [comment]
Hi there @begin-again and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by begin-again** [comment, marked as answer]
This has been working again as of late
